### PR TITLE
Finance complete dashboard

### PR DESCRIPTION
**Pull request by Stephek**
Text change

### CODE REVIEW

**Comment by lazcool** [comment]
This PR contains...
[CON-542 PR](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1674), which was approved, but has subsequent unreviewed changes
[CON-544 PR](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1666), which was approved, but has subsequent unreviewed changes
CON-568, branch (containing UI changes) - seems no PR
CON-550, branch - seems no PR
A couple of commits on this branch

File: src/SFA.DAS.EmployerFinance.Web/Views/EmployerAccountTransactions/Index.cshtml [hunk 1]
```diff
@@ -14,101 +14,159 @@
...
     </div>
+     @if (Model.Data.ExpiringFunds.HasValue && Model.Data.ExpiringFunds > 0 && Model.Data.ExpiryDate.HasValue)
+     {
+    <div class="column-one-third">
+        @{
+            var expiringFunds = Model.Data.ExpiringFunds.Value.ToString("C0", culture);
+
+            var expiringFundsClass = string.Empty;
+
+            if (expiringFunds.Length > 8)
```
Comment (lazcool) [inline src/SFA.DAS.EmployerFinance.Web/Views/EmployerAccountTransactions/Index.cshtml]: We could introduce a little helper function for deciding between the long and longer class names.

**Comment by lazcool** [comment]
Also contains https://skillsfundingagency.atlassian.net/browse/CON-707

**Comment by cofaulco** [comment]
Closing stale PR

**Pull request closed without merging**